src/webapp/tests/sana_test_case.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from webapp.models import *
+
+class SanaTestCase(TestCase):
```

Comment:
This class should be renamed to be something more specific. Not all tests can assume there's a user/procedure/page